ivadomed/scripts/curate_data_demo.py

```diff
@@ -0,0 +1,224 @@
+"""
+The data are structure in a folder such as:
+dataset/
```

Comment:
Should we make this dataset available alongside the script?

Reply:
yeah, people would love that

Reply:
I can create a dataset that is empty similar to the ones on open neuro.

Reply:
added demo dataset here: https://github.com/ivadomed/ivadomed/pull/1097/commits/29bccd6c2ff57adaf7bb4d06136191b6665ed5a2

Reply:
We usually don't put data in the main repo.
I would suggest to make a repo for the demo datatset.
Maybe rename it with as `demo_curation_microscopy_dataset` so it is not mistaken for a BIDS-compliant dataset.

We could add it to the [download_data.py](https://github.com/ivadomed/ivadomed/blob/1a3d193922938dc35e9eb77d6af7fd8e6d1bacfa/ivadomed/scripts/download_data.py#L19) scripts and add instructions to download it at the top of the curation script.